src/view/base-template.twig

```diff
@@ -93,7 +93,7 @@
     </div>
     {% endif %}
   </header>
-  <main id="main-container" class="{% if request.vocabid == '' or global_search %} frontpage{% elseif concept_uri != '' %} termpage{% elseif request.vocab != '' %} vocabpage{% if list_style %} {{ list_style }}{% endif %}{% endif %} py-5">
+  <main id="main-container" class="{% if request.vocabid == '' or global_search %}frontpage{% elseif concept_uri != '' %}termpage{% elseif request.vocab != '' %}{% if request.page == 'search'%}searchpage{% else %}vocabpage{% endif %}{% if list_style %} {{ list_style }}{% endif %}{% endif %} py-5">
```

Comment:
Oof. This could really use some refactoring... We already have a Twig variable `pageType` that I think would be a much better source of information for this, and perhaps it could be used directly as a CSS class name (adding some prefix). Oh well, maybe that's out of scope for this PR...

Reply:
That might be worth a shot, I'll have a look as generating the class name.